doc/api/crypto.rst

```diff
@@ -90,6 +90,11 @@ Private keys
 Certificate revocation lists
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. py:function:: dump_crl(type, crl)
```

Comment:
please use autodoc here.  currently the docstring is more verbose than the documentation. :)